mrp_routing/views/mrp_bom_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+
+        <record id="mrp_bom_form_view" model="ir.ui.view">
+            <field name="name">mrp.bom.form.view</field>
+            <field name="model">mrp.bom</field>
+            <field name="inherit_id" ref="mrp.mrp_bom_form_view" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='operation_ids']" position="before">
+                    <label for="routing_id" />
+                    <field name="routing_id" />
+                </xpath>
+                <xpath expr="//field[@name='operation_ids']" position="attributes">
+                    <attribute
+                        name="attrs"
+                    >{'invisible': [('type','not in',('normal','phantom'))], 'readonly': [('routing_id', '!=', False)]}</attribute>
```

Comment:
And i think that: 'readonly': [('routing_id', '!=', False)]} it could be deleted, because maybe you want to use a predefined  operation, but you want to change something or add another operation.
![Captura de pantalla de 2022-02-07 09-13-32](https://user-images.githubusercontent.com/90242955/152750002-136564e1-c3c6-4ee1-8d06-92e1d8727bee.png)

In this case, for example I want to use the routing of manufacturing a table, but i don't want to paint the table,  but for the readonly, Im not able to do that change.

Reply:
Also if is readonly, and you select an predefined operations, there is no way to deselect the routing_id.

Reply:
Done!, thanks for review, please check